index.html

```diff
@@ -23,13 +24,40 @@
         <div class="mode">
             <p class="mode__type">MODE: Who is this character?</p>
             <div class="mode__rules">
-                <p class="mode__title"><i class="fas fa-graduation-cap"></i> Mode rules</p>
+                <p class="mode__title"><i class="fas fa-graduation-cap"></i><span> Mode rules</span></p>
                 <p class="mode___description">You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select who from Star Wars is showed on the left (Jar Jar Binks right now) from available options.</p>
             </div>
+            <div class="mode__rules mode__hall">
+                <p class="mode__title"><i class="fas fa-id-badge"></i><span> Mode ranking</span></p>
+                <div class="ranking__table">
```

Comment:
skoro ranking__table to czemu nie użyłaś tabeli? :) albo display: table w css
+ jeśli chcemy się trzymać danego widoku z figmy to tam tekst nie jest wyśrodkowany i między wynikiem punktowym a Playerem jest odstęp

Reply:
pytanie czy chcemy.... jak dla mnie to widok z Figmy jest brzydszy (razi tyle wolnej przestrzeni w drugiej kolumnie). co do table to pamiętam z tych tutoriali, że zawsze przestrzegali przed table, że to zło i że lepiej zawsze sięgać po grid przy bardziej złożonych tabelach.

Reply:
tak, przestrzegali przed tabalami do budowania layoutów, tak jak kiedyś się cała stronę po prostu wrzucało w niewidoczną tabelę, przynajmniej ja to tak rozumiem. tu jeśli mamy faktycznie tabelkę z wynikami to nic złego w tym nie widzę :) @ktprezes co sadzisz?

Reply:
dokładnie tak, z tą tabelą chodzi o to że jak kiedyś nie było jeszcze grida to się układało stronę za pomocą tabeli.